ci-operator/step-registry/cucushift/installer/rehearse/vsphere/ipi/disconnected/cucushift-installer-rehearse-vsphere-ipi-disconnected-workflow.yaml

```diff
@@ -5,5 +5,7 @@ workflow:
     - chain: cucushift-installer-rehearse-vsphere-ipi-disconnected-provision
     post:
     - chain: cucushift-installer-rehearse-vsphere-ipi-disconnected-deprovision
+    env:
+      PULL_THROUGH_CACHE: disabled
```

Comment:
Can you also update `cucushift-installer-rehearse-vsphere-ipi-disconnected-techpreview` together

Reply:
ok

Reply:
Also `cucushift-installer-rehearse-vsphere-upi-provision`

Reply:
Can we move these setting to provision chain ?